src/main/java/org/jenkins/plugins/statistics/gatherer/util/LogbackUtil.java

```diff
@@ -0,0 +1,44 @@
+package org.jenkins.plugins.statistics.gatherer.util;
+
+import ch.qos.logback.classic.Logger;
+import ch.qos.logback.classic.LoggerContext;
+import ch.qos.logback.classic.util.ContextInitializer;
+import ch.qos.logback.core.joran.spi.JoranException;
+
+import java.net.MalformedURLException;
+import java.net.URL;
+
+public class LogbackUtil {
+    private static final String STATISTICS_GATHERER_LOGGER = "statistics-gatherer";
+    private static Logger log;
```

Comment:
I'd rename this to `logger` instead of `log` for clarity

Reply:
True, let me rename it.